cellpack/autopack/writers/__init__.py

```diff
@@ -431,3 +435,108 @@ def save(
             self.save_as_simularium(env, seed_to_results_map)
         else:
             print("format output " + output_format + " not recognized (json,python)")
+
```

Comment:
Do we want this in `__init__.py`? I feel like this could be moved to a separate file `MarkdownWriter.py` in the same folder

Reply:
good point, fixed!